### PR TITLE
Added Back Button Addon

### PR DESCRIPTION
**Pull request by shinglyu**
Just migrated to the new arch

### CODE REVIEW

File: app/apps-v2.json [hunk 1]
```diff
@@ -46,5 +46,15 @@
       "type": "addon",
       "url": "https://github.com/mikehenrty/fxos-shb-plus",
       "revision": "9b4ed3b83735a3fcd764b317499efde5b1e64f68"
+    },
+    "https://shinglyu.github.io/fxos-back-button/manifest.webapp": {
```
Comment (mikehenrty) [inline app/apps-v2.json]: Let's change this to the update.manifest too, just for consistency.

File: app/apps-v2.json [hunk 2]
```diff
@@ -46,5 +46,15 @@
...
+      "manifestURL": "https://shinglyu.github.io/fxos-back-button/update.webapp",
```
Comment (mikehenrty) [inline app/apps-v2.json:91]: The `package_path` inside of your update.manifest still points to the old `package.zip`. Please make it point to the `new-package.zip` instead.

**Comment by shinglyu** [comment]
I just fixed them all and rebased to the latest master, thank you.

**Comment by mikehenrty** [comment]
In your `manifest.json` you have a field called `manifest_json`. What you want there is `manifest_version`, otherwise it won't install properly.

https://developer.chrome.com/extensions/manifest/manifest_version

**Comment by shinglyu** [comment]
Geeee, I'm so stupid. I just updated the `manifest.json` 

By the way, may I ask how you install the addon from a local PC? Do I need to host it?

**Comment by mikehenrty** [comment]
I've been installing my add-ons by building the hackerplace, only pulling the local apps-v2.json (not the hosted one), and then installing from hackerplace. If installing from webIDE is broken please file a bug :)

**Comment by mikehenrty** [comment]
Hrmm, after installing your add-on, I didn't see anything show up in the homebar. Are you able to test your add-on locally?

**Comment by shinglyu** [comment]
Actually I don't know how to test it locally, since the WebIDE doesn't work with the new manifest format. Can I temporarily change `update.webapp` to `manifest.webapp` and load it in WebIDE?

**Comment by mikehenrty** [comment]
Hi @shinglyu ,

Yeah unfortunately, we don't have support yet to test add-ons locally. But that should be coming soon.
https://bugzilla.mozilla.org/show_bug.cgi?id=1196734

In the meantime, see this thread for some alternatives to testing:
https://groups.google.com/d/msg/mozilla.dev.b2g/m10n9RpNwlI/utQ4-8JgEQAJ

**Comment by shinglyu** [comment]
@mikehenrty : I've fixed some manifest errors, now it works on the latest flame build (installed throught nightly WebIDE). Thank you.

**Comment by mikehenrty** [comment]
Hi @shinglyu, sorry about the delay.

The new manifest points to a package.zip that does not contain manifest.json. You will need to update your package to include this.

**Comment by shinglyu** [comment]
@mikehenrty : Sorry for the delay, I updated the zip and tried installed it with WebIDE, it should work now.

**Comment by mikehenrty** [comment]
Hi Shing, please submit your add-on to the Marketplace instead. We are deprecating the Hackerplace. Thanks!

**Pull request closed without merging**